modules/node_modules/@ncigdc/containers/RepositoryPage.js

```diff
@@ -120,40 +133,31 @@ export const RepositoryPageQuery = {
     files_size: null,
     files_sort: null,
     filters: null,
-    idAutocompleteCases: null,
-    runAutocompleteCases: false,
-    idAutocompleteFiles: null,
-    runAutocompleteFiles: false,
+    autocompleteTypes: [],
+    idAutocomplete: null,
+    runAutocomplete: false,
   },
   fragments: {
-    /*
     autocomplete: () => Relay.QL`
-      fragment on Repository {
-        cases {
-          hits(first: 5, id_autocomplete: $idAutocompleteCases) {
-            edges @include(if: $runAutocompleteCases) {
-              node {
-                case_id
-                project {
-                  project_id
-                }
+      fragment on Root {
+        query (query: $idAutocomplete types: $autocompleteTypes) @include(if: $runAutocomplete) {
```

Comment:
looks like can't do any JS in that `@include` directive besides a straight up boolean
```
Error: GraphQLError: Syntax Error RepositoryPage (2:93) Unexpected (

1: fragment RepositoryPage_ViewerRelayQL on Root {
2:         autocomplete: query (query: $idAutocomplete types: $autocompleteTypes) @include(if: ($idAutocomplete || '').length > 0) {
```